src/app/collection/pages/collection-ncyte/curators/curator-card/curator-card.component.scss

```diff
@@ -0,0 +1,11 @@
+div{
+    margin-right: 100px;
```

Comment:
I would be careful using margin to separate because it will need be changed often for different screen widths. There are other options like justify-content for display:flex; items.
Ex. 
```
<div class="container">
     <curator-card></curator-card>
     <curator-card></curator-card>
</div>
```
.container {
     display: flex;
     justify-content: space-evenly;
}